image_provisioner/playbooks/roles/clone-repos/tasks/main.yaml

```diff
@@ -29,13 +29,22 @@
     path: /root/openshift-ansible 
   register: openshift_ansible
 
+- name: get the openshift version
+  shell: echo "{{ openshift_version }}" |  awk -F '.' '{ print $1"."$2 }'
+  register: version
+
+- name: set openshift-ansible branch to be cloned
+  set_fact:
+     openshift_ansible_branch: release-{{ version.stdout }}
```

Comment:
`"{{ openshift_version.split('.')[0] }}.{{ openshift_version.split('.')[1] }}"` this will avoid having to add an additional shell task just to manipulate the string.